gulpfile.cjs

```diff
@@ -1599,7 +1599,9 @@ function createTypeScriptDefinitions() {
       /= "WebGLConstants\.(.+)"/gm,
       // eslint-disable-next-line no-unused-vars
       (match, p1) => `= WebGLConstants.${p1}`
-    );
+    )
+    // Strip const enums which can cause errors - https://www.typescriptlang.org/docs/handbook/enums.html#const-enum-pitfalls
+    .replace(/^(\s*)(export )?const enum (\S+) {(\s*)$/gm, "$1$2enum $3 {$4");
```

Comment:
This looks good to me. We probably could have gotten away with an even simpler `const enum` `enum` call, but the good news is that if this ever breaks the build-ts step will fail so it's fine either way.